### DISCUSSION TITLE
change text of "Server xyz uses an untrusted certificate [Certificate not valid. This is most probably a server issue.] - Wiki: https://sabnzbd.org/certificate-errors"

### DISCUSSION THREAD

**Discussion by sanderjo** [sabnzbd/sabnzbd#2276] [General]
The text

"Server xyz uses an untrusted certificate [Certificate not valid. This is most probably a server issue.] - Wiki: https://sabnzbd.org/certificate-errors"

was valid when we introduced it: the problem was on the newsserver side.

But those newsserver providers have improved their setup. And thus the statement "This is most probably a server issue" is not true anymore. Nowadays, it might more likely be a bad client setup.

So ... just remove that string "This is most probably a server issue"  from the message, and keep the URL in place?

**Comment by Safihre** [comment]
We could change it to a text similar to what Python/OpenSSL report: `Certificate verification failed`.

**Reply by sanderjo** [reply]
Good. I'll do that.
Translations will follow automatically?

**Reply by Safihre** [reply]
Yes!
